scripts/build.py

```diff
@@ -510,6 +515,16 @@ def build_third_party_libs(third_party_directory_path):
             dependency_filepaths['fonts'], FONTS_DIR))
 
 
+def webpack_build():
+    """Execute webpack build process."""
```

Comment:
Could you give a more detailed explanation of what the webpack build process actually does here (i.e. what files is it modifying and where is it putting them)? That will help devs reason about what's going on.

Reply:
Yes I will include the explanation.

Reply:
Explanation is still missing.

Reply:
Done.